jscripts/sceditor/editor_plugins/undo.js

```diff
@@ -0,0 +1,184 @@
+(function ($) {
```

Comment:
The file should probably be minified, at least other core SCEditor plugins are: https://github.com/mybb/mybb/blob/feature/jscripts/sceditor/editor_plugins/bbcode.js And, not connected with the PR, but format/xhtml plugin files can actually be removed btw since they're not used anywhere.